src/pages/ChefsPage/ChefsPage.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import styles from "./ChefsPage.module.scss";
+import data from "../../data/data.json";
+import { Chef } from "../../types/types";
+import ChefCard from "../../components/ChefCard/ChefCard";
+import useIsMobile from "../../hooks/useIsMobile";
+import useIsTablet from "../../hooks/useIsTablet";
+
+const ChefsPage = () => {
+  const chefs: Chef[] = data.data.chefs;
+  const isMobile = useIsMobile();
+  const isTablet = useIsTablet();
+  const isMobileOrTable = isMobile || isTablet;
+
+  enum IndexType {
+    ALL_INDEX = 0,
```

Comment:
`ALL_INDEX` is not a clear name. Next time maybe name it `ALL_CHEFS_TAB_INDEX`

Reply:
@nitzanpap  done